src/styles/index.scss

```diff
@@ -1,3 +1,8 @@
+@import 'blocks/card';
+@import 'blocks/stars';
+
```

Comment:
reset default margin for `p` to avoid resetting this value for each class selector